faw/pdf-observatory/main.py

```diff
@@ -40,7 +37,7 @@
 app_hostname = None
 app_hostport = None
 
-app_init = None
+app_init = None  # Looks unused
```

Comment:
Not sure how this looks unused? It holds onto asynchronous futures in a few places.